test/JDBC/upgrade/14_8/schedule

```diff
@@ -398,4 +398,5 @@ datetime2fromparts
 timefromparts
 orderby-before-15_3
 BABEL-3215
-BABEL-4078-before-14_8-or-15_3
+BABEL-4078
```

Comment:
This change is expected, `BABEL-4078-before-14_8-or-15_3` test meant for before 14_8 release.